card/src/main/java/com/adyen/checkout/card/internal/ui/DefaultCardDelegate.kt

```diff
@@ -733,6 +730,19 @@ internal class DefaultCardDelegate(
         }
     }
 
+    private fun getCardBrand(stateOutputData: CardOutputData): String? {
```

Comment:
It looks like only `detectedCardTypes` is used from the output data. Only passing that would look a bit cleaner